turtlestar-evaluationtest.py

```diff
@@ -0,0 +1,68 @@
+import pytest
+
+from pathlib import Path
+from shutil import copyfile
+from tempfile import TemporaryDirectory
+
+from rdflib.exceptions import ParserError
+
+from rdflib import Graph
+from rdflib.util import guess_format
+
+
+from rdflib.plugin import register
+from rdflib.parser import Parser
+from rdflib.serializer import Serializer
+
+import rdflib
+from rdflib import URIRef
+from rdflib.namespace import RDF
+from rdflib.namespace import FOAF
+
+g = Graph()
+g.parse(data="test/turtlestar-evaluation/turtle-star-eval-01.ttl", format = "ttls")
+print(g.serialize(format = "ttlstar"))
```

Comment:
test should not be bare in the file, it should be inside test functions so that pytest correctly reports on it.

Further, this should ideally be using a parameterized test, most likely the right thing to do here is to use [`test/test_roundtrip.py`](https://github.com/RDFLib/rdflib/blob/main/test/test_roundtrip.py)

Probably a good baseline is to look at the n3 suite roundtrips being done:

https://github.com/RDFLib/rdflib/blob/3a418218d6bcdb46f78342e14c024063e2f53e71/test/test_roundtrip.py#L490-L504